### PR TITLE
✨️ : gitlab module import

### PR DESCRIPTION
**Pull request by juwit**

### CODE REVIEW

File: src/main/resources/templates/modules_import.html [hunk 1]
```diff
@@ -76,19 +76,19 @@
...
-<!--                </b-button>-->
-
-<!--                <repository-import v-if="selected == 'gitlab'" :registry="'gitlab'"></repository-import>-->
-
-<!--                <template v-slot:footer>-->
-<!--                    <em>Import an existing module code from existing Gitlab repository</em>-->
-<!--                </template>-->
-<!--            </b-card>-->
-<!--        </div>-->
+        <div class="col-md-4" th:if="${user.oAuth2User.provider == 'gitlab'}">
```
Comment (cdubuisson) [inline src/main/resources/templates/modules_import.html]: Is it possible to extract a common vue component for github/gitlab import ?

File: src/main/java/io/codeka/gaia/registries/controller/GitlabRegistryController.kt [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        module.gitBranch = "master"
+        module.name = gitlabRepository.fullName
+        module.cliVersion = cliRepository.listCLIVersion().first()
+
+        module.registryDetails = RegistryDetails(RegistryType.GITLAB, gitlabRepository.id)
+        module.createdBy = user
+
+        // get variables
+        val variablesFile = gitlabRegistryApi.getFileContent(user, projectId, "variables.tf")
+        module.variables = hclParser.parseVariables(variablesFile)
```
Comment (cdubuisson) [inline src/main/java/io/codeka/gaia/registries/controller/GitlabRegistryController.kt]: It should be good to make variables editable by default

File: src/main/resources/templates/modules_import.html (hunk 1 above)
Comment (juwit) [reply src/main/resources/templates/modules_import.html]: done ✔️

File: src/main/java/io/codeka/gaia/registries/controller/GitlabRegistryController.kt (hunk 2 above)
Comment (juwit) [reply src/main/java/io/codeka/gaia/registries/controller/GitlabRegistryController.kt]: done ✔️

**Pull request merged**